content/W2015/W15-blog-caleybrock.md

```diff
@@ -28,15 +28,15 @@ Although her peers did not outright discourage her, they were condescending. Cal
 
 "The only reason I kept going was to prove to myself that I could keep going," she explains.
 
-With perseverance, Caley successfully transitioned into the computer science co-op program.  She had to complete five consecutive study terms in order to catch up with her class, which damaged her confidence.
+With perseverance, Caley successfully transitioned into the computer science co-op program. She had to complete five consecutive study terms in order to catch up with her class, which damaged her confidence.
 
-Caley found more confidence and support in her work terms. She believes that employers value what you can offer more than "how long you’ve programmed for" or what "grades you have".
+Caley found more confidence and support in her work terms. She believes that employers value what you can offer more than "how long you've programmed for" or what "grades you have".
 
 "Anybody can learn to program," she elaborates, "your value is what you can do top of that."
```

Comment:
"elaborates" => "claims"?

I would change the punctuation to `she claims. "Your`